### PR TITLE
partner_tlr_update AFC-1382: fixed the loading of tlr data by hardcod…

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**
…ing all necessary namespaces instead of using nsmap.

### CODE REVIEW

File: partner_tlr_update/models/res_partner.py [hunk 1]
```diff
@@ -89,19 +89,32 @@ def update_tlr_data(self):
                 response = api_client.get_tjansteleverantor(self.legacy_no)
                 if response.status_code == 200:
                     self.parse_xml_tjansteleverantor_data(response.text)
-                response = api_client.get_utforande_verksamhet_id(
-                    self.legacy_no)
+                    _logger.info("PARSED XML")
```
Comment (arturchilangwa-vertel) [inline partner_tlr_update/models/res_partner.py:92]: Do we need all the _loggers for debbuging in pre-production enviorment?

File: partner_tlr_update/models/res_partner.py [hunk 2]
```diff
@@ -119,35 +133,43 @@ def update_from_xml(self, xml, match_name):
...
     def parse_xml_organisationsnummer_data(self, xml):
+        #_logger.info("XML: %s" % xml)
         root = etree.fromstring(self.clearing_xml(xml))
+        _logger.info("ROOT: %s" % root)
         self.update_subsidiary(root)
 
     @api.multi
     def update_contact_person(self, xml):
-        person_id = xml.find(".//ns25:kontaktpersonId", namespaces=xml.nsmap)
+        person_id = xml.find(".//ns25:kontaktpersonId", namespaces={'ns25':'http://arbetsformedlingen.se/datatyp/tjansteleverantor/kontaktperson/v17'})
```
Comment (arturchilangwa-vertel) [inline partner_tlr_update/models/res_partner.py:163]: should this be hardcoded?

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

File: partner_tlr_update/models/res_partner.py (hunk 2 above)
Comment (SebastianGieser-Vertel) [reply partner_tlr_update/models/res_partner.py:163]: Preferably not, but xml.nsmap only returns the namespace of the root tag so this is the best I could come up with.

File: partner_tlr_update/models/res_partner.py (hunk 1 above)
Comment (SebastianGieser-Vertel) [reply partner_tlr_update/models/res_partner.py:92]: Might be helpful if something goes wrong even if it's maybe a bit excessive currently, they should be removed once it's tested and working.
It's not like it'll spam the log constantly, and even when it runs the logging isn't spitting out anything too big.

**Review by daerio — APPROVED** [review]

**Pull request merged**